src/isc-dhcp/patch/0007-If-destination-of-BOOTREQUEST-is-directed-broadcast-.patch

```diff
@@ -12,52 +12,44 @@ Subject: [PATCH 3/3] If destination of BOOTREQUEST is directed broadcast,
  3 files changed, 131 insertions(+), 16 deletions(-)
 
 diff --git a/common/discover.c b/common/discover.c
-index 8e7f632..73eb8a9 100644
+index 98ac46a..53afecc 100644
 --- a/common/discover.c
 +++ b/common/discover.c
-@@ -227,6 +227,7 @@ struct iface_conf_list {
+@@ -236,6 +236,7 @@ struct iface_conf_list {
  struct iface_info {
  	char name[IF_NAMESIZE+1];	/* name of the interface, e.g. "bge0" */
  	struct sockaddr_storage addr;	/* address information */
-+    struct sockaddr_storage netmask;    /* netmask information */
++        struct sockaddr_storage netmask;    /* netmask information */
```

Comment:
ISC-DHCP source uses tabs for indenting. Leading whitespace here should be a single tab to align properly with the rest of the source.

Reply:
Done.